File: Data Structures & Algorithms/two-integer-sum/submission-0.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer, Integer> indexMap = new HashMap<>();
        for (int index = 0; index < nums.length; index++) {
            int number = nums[index];
            Integer match = indexMap.get(number);
            if (match == null) {
                indexMap.put(target - number, index);
            } else {
                return new int[]{match, index};
            }
        }
        return null;
    }
}

File: Data Structures & Algorithms/jump-game/submission-0.py
class Solution:
    def canJump(self, nums: List[int]) -> bool:
        index = len(nums) - 1
        for i in range(len(nums) - 2, - 1, - 1):
            if index - i <= nums[i]:
                index = i
        print(index)
        return index == 0